tests/test_metrics.py

```diff
@@ -60,110 +60,124 @@ def legacy_mean_dice(results, gt_seg_maps, num_classes, ignore_index):
 
 def test_metrics():
     pred_size = (10, 30, 30)
-    num_classes = 19
     ignore_index = 255
-    results = np.random.randint(0, num_classes, size=pred_size)
-    label = np.random.randint(0, num_classes, size=pred_size)
-    label[:, 2, 5:10] = ignore_index
-    all_acc, acc, iou = eval_metrics(
-        results, label, num_classes, ignore_index, metrics='mIoU')
-    all_acc_l, acc_l, iou_l = legacy_mean_iou(results, label, num_classes,
-                                              ignore_index)
-    assert all_acc == all_acc_l
-    assert np.allclose(acc, acc_l)
-    assert np.allclose(iou, iou_l)
-
-    all_acc, acc, dice = eval_metrics(
-        results, label, num_classes, ignore_index, metrics='mDice')
-    all_acc_l, acc_l, dice_l = legacy_mean_dice(results, label, num_classes,
-                                                ignore_index)
-    assert all_acc == all_acc_l
-    assert np.allclose(acc, acc_l)
-    assert np.allclose(dice, dice_l)
-
-    all_acc, acc, iou, dice = eval_metrics(
-        results, label, num_classes, ignore_index, metrics=['mIoU', 'mDice'])
-    assert all_acc == all_acc_l
-    assert np.allclose(acc, acc_l)
-    assert np.allclose(iou, iou_l)
-    assert np.allclose(dice, dice_l)
-
-    results = np.random.randint(0, 5, size=pred_size)
-    label = np.random.randint(0, 4, size=pred_size)
-    all_acc, acc, iou = eval_metrics(
-        results,
-        label,
-        num_classes,
-        ignore_index=255,
-        metrics='mIoU',
-        nan_to_num=-1)
-    assert acc[-1] == -1
-    assert iou[-1] == -1
-
-    all_acc, acc, dice = eval_metrics(
-        results,
-        label,
-        num_classes,
-        ignore_index=255,
-        metrics='mDice',
-        nan_to_num=-1)
-    assert acc[-1] == -1
-    assert dice[-1] == -1
-
-    all_acc, acc, dice, iou = eval_metrics(
-        results,
-        label,
-        num_classes,
-        ignore_index=255,
-        metrics=['mDice', 'mIoU'],
-        nan_to_num=-1)
-    assert acc[-1] == -1
-    assert dice[-1] == -1
-    assert iou[-1] == -1
+    max_classes = 255
+    for num_classes in range(6, max_classes):
```

Comment:
Why testing from 6 to 255?

Reply:
I want to expand the test range as wider as possible.